### ISSUE TITLE
Nagrand ogres giving incorrect reputation when in a group

### ISSUE THREAD

**Issue by Nergigante** [dalaranwow/dalaran-wow#5709]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Killing ogres in nagrand only grants the player 5 reputation with Kurenai when in a group with another player.  It grants 10 reputation when killing them solo.  
**Expected behaviour**: Tell us what should happen instead
Killing ogres in nagrand should always give 10 reputation, even if someone is in a group.
**Steps to reproduce the problem**:

1. Group up with another player that is within range to receive experience and loot for a kill.
2. Kill warmaul ogres in nagrand, any will do.
3. Notice that they are giving 5 reptuation.  Repeat 2 while outside of the group and notice they are giving the full 10.

In addition, after researching this issue on the bugtracker, I see that this issue has persisted in various forms since 2014.  One post claimed that the ogres were giving them 5 rep as well.  However, they felt it was due to their level in relation to the mobs.  I have not found this to be the case, though I am getting only 5 rep while in a group just like them.

**Include proofs for this behaviour**
Screenshot is attached.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
npc=17134
npc=17135

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Nergigante** [comment]
![WoWScrnShot_032719_221327](https://user-images.githubusercontent.com/49010516/55124900-4f9a8800-50de-11e9-97b6-e982452a6c0e.jpg)
![WoWScrnShot_032719_221232](https://user-images.githubusercontent.com/49010516/55124916-5a551d00-50de-11e9-9fe2-97e4738586d6.jpg)
